log_queries.py

```diff
@@ -81,3 +88,28 @@ def log_query(service, question: str, answer: str, sentiment: str = "N/A", sprea
         return 1
 
     return 0
+
+@app.route("/log/query", methods=["POST"])
+def log_route():
+    request_body = request.get_json()
+    question = request_body.get("question")
+    answer = request_body.get("answer")
+    sentiment = request_body.get("sentiment")
+    request_body = request.get_json()
+    question = request_body.get("question")
+    answer = request_body.get("answer")
+    sentiment = request_body.get("sentiment")
+    if question is None or answer is None or sentiment is None:
+        return "Request was missing a required parameter", BAD_REQUEST
+    if(log_query(config_api(), question, answer, sentiment) == 0):
+        return "Success", SUCCESS
+    else:
+        return "Failed to log the query", SERVER_ERROR
+
+
+def get_spreadsheet_id():
```

Comment:
It would be better to rely on env variables for this, but we don't have a deployment process that can create those right now. This should be supported though by the environ.get() above for the future.